ketcher-autotests/tests/API/api-set-get-molecule.spec.ts

```diff
@@ -54,7 +54,7 @@ test.describe('Tests for API setMolecule/getMolecule', () => {
     */
     await waitForSpinnerFinishedWork(
       page,
-      async () => await setMolecule(page, 'C1C=CC=CC=1'),
+      async () => await setMolecule(page, 'C1C=CC=CC=1', false),
```

Comment:
Does it make sense to pass `false` to these functions in any of autotests?
These tests currently cover the default behaviour with rescaling and now it leads for these autotests to fail on the pipeline